### PR TITLE
Fixed a small typo on the program name

### PR DESCRIPTION
**Pull request by mssola**
This was found when printing out the version of the program.

Signed-off-by: Miquel Sabaté Solà <msabate@suse.com>

### CODE REVIEW

File: catatonit.c [hunk 1]
```diff
@@ -102,7 +102,7 @@ static void help(void)
 
 static void version(void)
 {
-	fprintf(stdout, "tini version %s_%s\n", PROGRAM_VERSION, PROGRAM_NAME);
+	fprintf(stdout, "%s version %s_%s\n", PROGRAM_NAME, PROGRAM_VERSION, PROGRAM_NAME);
```
Comment (vrothberg) [inline catatonit.c:105]: This was intentional to make `docker-info` happy. Docker is hard-coded against tini and this was the (unfortunate) hack to make it work nicely with catatonit.

Did Docker's behavior change?

Comment (mssola) [reply catatonit.c:105]: Well, I have no idea. I thought this was just a silly typo :smile:. If this is not the case, maybe I can add a comment explaining just that.

**Comment by cyphar** [comment]
Sadly this isn't a typo, as @vrothberg said this is a silly workaround for Docker hard-coding how they detect `tini`'s version. If you change it, Docker starts giving scary (and completely pointless) warnings.

**Comment by cyphar** [comment]
Ah right, it wouldn't be a bad idea to add a comment.

**Comment by mssola** [comment]
Added a comment with a text based on @vrothberg's explanation (thanks!).

**Comment by cyphar** [comment]
LGTM.

**Pull request merged**